jetbot/bot.py

```diff
@@ -498,10 +494,16 @@ async def channels(ctx: commands.Context):
                             category = await guild.create_category(item["category_name"])
                         categories[item["category_name"].lower().strip()] = category
 
-                    # Handle multi-channel configuration
+                    # Handle channels configuration
                     if "channels" in item:
                         for channel_config in item["channels"]:
-                            channel = await self.product_channel(channel_config["name"], item["category_name"])
+                            channel = None
+                            for existing in valid_channels:
+                                if existing.name == channel_config["name"] and existing.category and \
+                                   existing.category.name.lower().strip() == item["category_name"].lower().strip():
+                                    channel = existing
+                                    break
```

Comment:
I think here is where the perms/type logic you've added above should be instead?